src/components/charts/SavedValues/SavedValuesList.tsx

```diff
@@ -228,40 +243,67 @@ export default function SavedValuesList() {
                         renderEmptyState('No matching preset values', 'We couldn’t find any matching results', true)
                     ) : (
                         <>
-                            <div className="preset-values-row fw-6 cn-7 fs-12 border-bottom text-uppercase pt-8 pr-16 pb-8 pl-16">
-                                <div className="pr-16"></div>
-                                <div className="pr-16">Name</div>
-                                <div className="pr-16">Version</div>
-                                <div className="pr-16"></div>
+                            <div className="preset-values-row fw-6 cn-7 fs-12 border-bottom text-uppercase pt-8 pr-20 pb-8 pl-20">
+                                <div className="preset-values-row-item" />
```

Comment:
can we remove  className="preset-values-row-item" as we are not using preset-values-row-item class

Reply:
Done